addons/web/controllers/main.py

```diff
@@ -1184,12 +1184,16 @@ def get_fonts(self, fontname=None):
 
         fonts = []
         if fontname:
-            font_path = get_resource_path('web', 'static/src/fonts/sign', fontname)
-            if not font_path:
-                return []
-            font_file = open(font_path, 'rb')
-            font = base64.b64encode(font_file.read())
-            fonts.append(font)
+            module_path = get_module_path('web')
+            fonts_folder_path = module_path + '/static/src/fonts/sign/'
+            module_resource_path = get_resource_path('web', 'static/src/fonts/sign/' + fontname)
+            if fonts_folder_path and module_resource_path:
+                fonts_folder_path = os.path.join(os.path.normpath(fonts_folder_path), '')
+                module_resource_path = os.path.normpath(module_resource_path)
+                if module_resource_path.startswith(fonts_folder_path):
+                    font_file = file_open(module_resource_path, 'rb')
```

Comment:
This file handler isn't closed after its usage.
You should use ```with .. as:``` statements so you don't forget to close it :P 

```suggestion
                    with file_open(module_resource_path, 'rb') as font_file:
```

Reply:
Thanks, I changed both